### PR TITLE
Release 3.0.0

### PR DESCRIPTION
**Pull request by rannick**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/rnafusion/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/rnafusion _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Review by apeltzer — APPROVED** [review]

File: nextflow.config [hunk 1]
```diff
@@ -295,7 +290,7 @@ manifest {
     description     = """Nextflow rnafusion analysis pipeline, part of the nf-core community."""
     mainScript      = 'main.nf'
     nextflowVersion = '!>=23.04.0'
-    version = '2.4.0'
+    version         = '3.0.0dev'
```
Comment (fevac) [inline nextflow.config:293]:
```suggestion
    version         = '3.0.0'
```

File: tower.yml [hunk 2]
```diff
@@ -13,20 +13,16 @@ reports:
     display: "FusionInspector TSV report"
   "**/fusionreport/*/*_fusionreport_index.html":
     display: "Fusion-report HTML report"
-  "**/megafusion/*_fusion_data.vcf":
+  "**/vcf_collect/*_fusion_data.vcf":
```
Comment (fevac) [inline tower.yml:16]:
```suggestion
  "**/vcf/*_fusion_data.vcf.gz":
```

File: assets/multiqc_config.yml [hunk 3]
```diff
@@ -1,13 +1,37 @@
 report_comment: >
-  This report has been generated by the <a href="https://github.com/nf-core/rnafusion/2.4.0" target="_blank">nf-core/rnafusion</a>
+  This report has been generated by the <a href="https://github.com/nf-core/rnafusion/releases/tag/dev" target="_blank">nf-core/rnafusion</a>
```
Comment (fevac) [inline assets/multiqc_config.yml:2]:
```suggestion
  This report has been generated by the <a href="https://github.com/nf-core/rnafusion/releases/tag/3.0.0" target="_blank">nf-core/rnafusion</a>
```

File: assets/multiqc_config.yml [hunk 4]
```diff
@@ -1,13 +1,37 @@
...
   analysis pipeline. For information about how to interpret these results, please see the
-  <a href="https://nf-co.re/rnafusion/2.4.0/output" target="_blank">documentation</a>.
+  <a href="https://nf-co.re/rnafusion/dev/docs/output" target="_blank">documentation</a>.
```
Comment (fevac) [inline assets/multiqc_config.yml:4]:
```suggestion
  <a href="https://nf-co.re/rnafusion/3.0.0/docs/output" target="_blank">documentation</a>.
```

**Review by fevac — APPROVED** [review]
🚀

File: CHANGELOG.md [hunk 5]
```diff
@@ -3,6 +3,43 @@
...
+- Build CRAM index in the same directory as CRAM files for arriba and STAR-Fusion [#427](https://github.com/nf-core/rnafusion/pull/427)
+
+### Changed
+
+- Replace PICARD_MARKDUPLICATES with GATK4_MARKDUPLICATES [#409](https://github.com/nf-core/rnafusion/pull/409)
+- Removed `--fusioninspector_filter` and `--fusionreport_filter` in favor of `--tools_cutoff` (default = 1, no filters applied) [#389](https://github.com/nf-core/rnafusion/pull/389)
+- Now publishing convert2bed output to convert2bed to keep the output file for mosdepth [#420](https://github.com/nf-core/rnafusion/pull/420)
+- No more checks for existence of samplesheet, which made building references fail (building references uses a fake sample sheet if none is provided) [#420](https://github.com/nf-core/rnafusion/pull/420)
+- `--extreme_sensitivity` used for fusioninspector to minimize fusioninspector filtering [#424](https://github.com/nf-core/rnafusion/pull/424)
+- `--extreme_sensitivity` removed in favor of `--max_sensitivity --max_mate_dist 10000000 --annotate --examine_coding_effect` to collect more data from fusioninspector [#426](https://github.com/nf-core/rnafusion/pull/426)
```
Comment (fevac) [inline CHANGELOG.md:20]: clean up

**Pull request merged**